modules/developer_manual/pages/core/unit-testing.adoc

```diff
@@ -97,63 +99,45 @@ If you use ownCloud functions in your class under test (i.e:
 OC::getUser()) you’ll need to bootstrap ownCloud or use dependency
 injection.
 
-You’ll most likely run your tests under a different user than the Web
-server. This might cause problems with your PHP settings (i.e.,
-`open_basedir`) and requires you to adjust your configuration.
+[NOTE]
+====
+You’ll most likely run your tests under a different user than the Web server. 
+This might cause problems with your PHP settings (i.e., `open_basedir`) and requires you to adjust your configuration.
+====
 
-An example for a simple test would be:
-
-/srv/http/owncloud/apps/myapp/tests/testaddtwo.php
+Given the class ``MyClass`` in your app:
```

Comment:
Is there a reason for having double ticks ?

Reply:
They slipped in by accident. I'll fix it up.